compiler/hash-tir/src/new/tuples.rs

```diff
@@ -41,7 +41,7 @@ pub struct TupleTerm {
 pub struct TuplePat {
     /// The original tuple type, if known or given as part of the literal (might
     /// contain holes).
-    pub original_ty: Option<TupleTy>,
+    // pub original_ty: Option<TupleTy>,
```

Comment:
why are these commented out?

Reply:
Yeah, i removed it cause it is not needed anymore (inference will store the result in cache if needed). Added commit to remove commented lines.